src/tests/ui/planner/toolbarHeader.ts

```diff
@@ -70,4 +71,10 @@ export class ToolbarHeader extends BaseElement {
     await this.titleTextInput.enterText(title);
     await this.saveFilterBtn.clickWhenReady();
   }
+  
+  async hasFilterCondition(labelName: string) {
```

Comment:
Can you please change this function to `getFilterConditions` and then use it as `expect(getFilterConditions()).toContain(...)` ?
Debugging `expect(hasFilterCondition(...)).toBeTruthy()` is difficult.

Reply:
@jarifibrahim Why is it difficult to debug `expect(hasFilterCondition(...)).toBeTruthy()`? It seems pretty straight forward boolean check. Can you please explain the problem you're facing with it and how `expect(getFilterConditions()).toContain(...)` can help solve it? We already have a var `activeFilterList` which basically is what i suppose will be returned by `getFilterConditions`.

Reply:
@divyanshiGupta so if you use expect(hasFilterCondition(...)).toBeTruthy() and the test fails, you will get message: Expected false to be true 
but if you use expect(getFilterConditions()).toContain(...) it will fail with the message 
Expected['label1', 'label2', 'Label3'] to contain label, so it will be easy to debug

Reply:
Got it. Thanks @Raunak1203 for the clarification on it. So, if i understand this correctly `getFilterConditions()` will return a list of all active filter conditions which we already have in `activeFilterList`. So, returning this var will be sufficient i guess?